src/main/scala/com/cognite/spark/datasource/EventsRelation.scala

```diff
@@ -82,6 +82,7 @@ object UpdateEventItem {
 }
 case class SourceWithResourceId(id: Long, source: String, sourceId: String)
 case class EventConflict(duplicates: Seq[SourceWithResourceId])
+case class EventFilter(_type: Option[String], subtype: Option[String])
```

Comment:
Personally I like 
```
`type`
```
better than `_type` but that's subjective.

Reply:
Looks like we've used `type` elsewhere as well, so I'll stick to that.